x-pack/platform/plugins/shared/stack_connectors/public/connector_types/xsoar/xsoar.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { GenericValidationResult } from '@kbn/triggers-actions-ui-plugin/public/types';
+import { XSOARConnector } from './types';
+import { XSOAR_CONNECTOR_ID, SUB_ACTION, XSOAR_TITLE } from '../../../common/xsoar/constants';
+import { ExecutorParams } from '../../../common/xsoar/types';
+import * as i18n from './translations';
+
+interface ValidationErrors {
+  name: string[];
+}
+
+export function getConnectorType(): XSOARConnector {
+  return {
+    id: XSOAR_CONNECTOR_ID,
```

Comment:
@cnasikas To account for the "intermediate release" process, considering this is a new connector, is it enough to add `hideInUi` here:

```suggestion
  return {
    id: XSOAR_CONNECTOR_ID,
    hideInUi: true,
```

Or is there anything else that needs to be done?

Reply:
Yes, I think it should be enough.

Reply:
@brijesh-elastic can you add that?